### PR TITLE
[PM-9969] Papercut - Update component library storybook docs

### PR DESCRIPTION
**Pull request by jprusik**
## 🎟️ Tracking

PM-9969

## 📔 Objective

Add component import examples, clean up some html/formatting nits

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: libs/components/src/badge/badge.stories.ts [hunk 1]
```diff
@@ -31,9 +31,9 @@ export const Primary: Story = {
     props: args,
     template: `
       <span class="tw-text-main">Span </span><span bitBadge [variant]="variant" [truncate]="truncate">Badge containing lengthy text</span>
-      <br><br>
+      <br /><br />
```
Comment (jprusik) [inline libs/components/src/badge/badge.stories.ts:34]: I used the prettier-preferred format for these tags

File: libs/components/src/dialog/dialog/dialog.mdx [hunk 2]
```diff
@@ -4,6 +4,10 @@ import * as stories from "./dialog.stories";
 
 <Meta of={stories} />
 
+```ts
+import { DialogService } from "@bitwarden/components";
+```
```
Comment (willmartian) [inline libs/components/src/dialog/dialog/dialog.mdx:9]:
```suggestion
```ts
import { DialogModule, DialogService } from "@bitwarden/components";
```
```

File: libs/components/src/badge/badge.mdx [hunk 3]
```diff
@@ -4,6 +4,10 @@ import * as stories from "./badge.stories";
 
 <Meta of={stories} />
 
+```ts
+import { CardComponent } from "@bitwarden/components";
```
Comment (willmartian) [inline libs/components/src/badge/badge.mdx:8]:
```suggestion
import { BadgeModule } from "@bitwarden/components";
```

File: libs/components/src/dialog/simple-dialog/simple-dialog.mdx [hunk 4]
```diff
@@ -4,6 +4,10 @@ import * as stories from "./simple-dialog.stories";
 
 <Meta of={stories} />
 
+```ts
+import { DialogService } from "@bitwarden/components";
```
Comment (willmartian) [inline libs/components/src/dialog/simple-dialog/simple-dialog.mdx:8]: Module is necessary if you are authoring a dialog; service is necessary if you are triggering one

```suggestion
import { DialogModule, DialogService } from "@bitwarden/components";
```

**Review by willmartian — CHANGES_REQUESTED** [review]
Thanks for this! A few suggestions:

File: libs/components/src/dialog/dialog/dialog.mdx [hunk 5]
```diff
@@ -4,6 +4,10 @@ import * as stories from "./dialog.stories";
 
 <Meta of={stories} />
 
+```ts
+import { DialogService } from "@bitwarden/components";
```
Comment (willmartian) [inline libs/components/src/dialog/dialog/dialog.mdx:8]:
```suggestion
import { DialogModule, DialogService } from "@bitwarden/components";
```

File: libs/components/src/badge/badge.mdx (hunk 3 above)
Comment (jprusik) [reply libs/components/src/badge/badge.mdx]: Ah, bad copy-paste; thanks

**Review by willmartian — APPROVED** [review]
:shipit: Thanks!

File: libs/components/src/dialog/simple-dialog/simple-dialog.mdx (hunk 4 above)
Comment (jprusik) [reply libs/components/src/dialog/simple-dialog/simple-dialog.mdx]: 💡

**Pull request merged**